source/exe/main_common.h

```diff
@@ -85,6 +85,8 @@ class MainCommonBase {
 private:
   void configureComponentLogLevels();
   void configureHotRestarter(Random::RandomGenerator& random_generator);
+
+  Thread::MainThread main_thread_;
```

Comment:
can you add a comment here why we declare it here? I think it's somewhat non-obvious

Reply:
done